prepare-toolchain

```diff
@@ -51,6 +51,7 @@ PLIST_PREFIX="${STAGING}/libplist-install"
 mkdir -p "${PLIST_PREFIX}"
 LC_ALL=C ./autogen.sh --prefix="${PLIST_PREFIX}" --without-cython PACKAGE_VERSION=latest
 make -j "${JOBS}" install
+cp -av "${PLIST_PREFIX}/bin/plistutil" "${INSTALLPREFIX}/bin"
```

Comment:
is `plistutil` statically linked to libplist? If not we'll either need to install libplist into the prefix (ensuring that the linkage is via `$ORIGIN`) or change it to link statically. In the former case we'll probably want to make `ldid` link against it dynamically as well.

Reply:
Not by default. Appears to be after passing `--enable-static --disable-shared` to autogen.